docs/appendix.rst

```diff
@@ -177,16 +204,16 @@ a single invocation of an easy-to use method
 Profile implementations in a WBEM server are not entirely free when making a
 choice of which methodology to implement:
 
-* Autonomous profiles in a WBEM server must implement the central class
-  methodology, and may in addition implement the new GetCentralInstances
+* Autonomous profiles in a WBEM server must implement the central-class
+  methodology, and may in addition implement the get-central-instances
```

Comment:
In DSP1033 1.1, it is actually called "GetCentralInstances methodology".

Reply:
Fixed DONE